src/content/docs/codestream/how-use-codestream/performance-monitoring.mdx

```diff
@@ -25,78 +25,311 @@ import codeLevelMetrics from 'images/code-level-metrics.png'
 
 import codeLevelCharts from 'images/code-level-charts.png'
 
-It's important to know how your code is performing in production and whether or not it's generating errors. To help you with this, New Relic CodeStream brings performance monitoring right into your IDE.
+import pixieDynamicLogging from 'images/pixie_dynamic_logging.png'
 
-## Connect CodeStream and New Relic One [#connect]
+import associateRepos from 'images/associate_repos.png'
 
-In order to get the most out of New Relic CodeStream, connect CodeStream to New Relic One via your [New Relic user key](/docs/apis/intro-apis/new-relic-api-keys/#user-api-key). Once done, for entities you're monitoring with New Relic One, you'll see your errors directly in CodeStream.
+import entitySelection from 'images/entity_selection.png'
+
+New Relic CodeStream allows developers to see how the code they're responsible
+for is performing in production by bringing observability into the IDE and
+making it part of their daily routine.
+
+## Connect CodeStream and New Relic [#connect]
+
+In order to access observability in your IDE you'll need to connect CodeStream
+to New Relic via your [New Relic user
+key](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher) (learn
+more about [New Relic user
+keys](/docs/apis/intro-apis/new-relic-api-keys/#user-api-key)). 
 
 <img
-  title="The unconnected observability section where you add your New Relic user key."
-  alt="A screenshot of the unconnected observability section where you add your New Relic user key."
+  title="Connecting CodeStream to New Relic with your New Relic user key."
+  alt="A screenshot of connecting CodeStream to New Relic with your New Relic user key"
   src={observabilitySectionConnect1}
```

Comment:
```suggestion
  src={observabilitySectionConnected1}
```

I think your build might have been failing because this image name was incorrect @planteater